examples/dapperish/dapper.go

```diff
@@ -63,20 +62,20 @@ func (d *DapperishTraceContext) NewChild() (opentracing.TraceContext, opentracin
 }
 
 // SetTraceTag complies with the opentracing.TraceContext interface.
-func (d *DapperishTraceContext) SetTraceTag(key, val string) opentracing.TraceContext {
+func (d *DapperishTraceContext) SetTraceTag(caseInsensitiveKey, val string) opentracing.TraceContext {
```

Comment:
I'd prefer we elaborate on this somewhere, even if this is a separate issue. For example [http/2 treats transmission of mixed case keys an error](https://http2.github.io/http2-spec/#HttpHeaders), even if comparison (at app layer) is permitted to be insensitive.


> Just as in HTTP/1.x, header field names are strings of ASCII characters that are compared in a case-insensitive fashion. However, header field names MUST be converted to lowercase prior to their encoding in HTTP/2. A request or response containing uppercase header field names MUST be treated as malformed (Section 8.1.2.6).